src/interface/src/app/plan/plan-table/plan-table.component.ts

```diff
@@ -60,12 +60,15 @@ export class PlanTableComponent implements AfterViewInit, OnInit {
       .listPlansByUser(null)
```

Comment:
Should we add a comment for this method that it sorts the plans by last created? I wonder if it makes more sense to sort by last updated once that's available?

Reply:
Good idea!